src/manage_nsfs/manage_nsfs_cli_responses.js

```diff
@@ -18,17 +19,19 @@ class ManageCLIResponse {
     /**
      * @param {ManageCLIResponseSpec} response_spec 
      */
-    constructor({ code, status, list }) {
+    constructor({ code, status, list, message }) {
         this.code = code;
         this.http_code = 200;
         this.status = status;
         this.list = list;
+        this.message = message;
     }
 
     to_string(detail) {
         const json = {
             response: {
                 code: this.code,
+                message: detail.name ? `${this.message}: ${detail.name}` : this.message,
```

Comment:
does adding the detail after the message makes sense in all cases?
and also, do we always have detail and detail.name?

Reply:
whenever we want to print the name of the object in the operation then only we are adding detail at the end of message otherwise only the set message would be printed for every operation

Reply:
the issue I see here is with accessing directly to detail.name while detail can be undefined

Reply:
Updated to optional chain. Thanks